sdk/inc/azure/iot/internal/az_iot_adu_internal.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/*
+ * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
+ * prefixed with an underscore ('_') directly in your application code. These symbols
+ * are part of Azure SDK's internal implementation; we do not document these symbols
+ * and they are subject to change in future versions of the SDK which would break your code.
+ */
```

Comment:
Since it is an internal file, consider this a nit (but add a double star to start, along with `@file` and a `@brief` for consistency with all other files in the repo):
```suggestion
/**
 * @file
 *
 * @brief Defines internal macros used by the ADU client.
 *
 * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
 * prefixed with an underscore ('_') directly in your application code. These symbols
 * are part of Azure SDK's internal implementation; we do not document these symbols
 * and they are subject to change in future versions of the SDK which would break your code.
 */
```

Reply:
https://github.com/Azure/azure-sdk-for-c/pull/2414